applications/ecallmgr/src/ecallmgr_fs_conference.erl

```diff
@@ -196,6 +196,7 @@ handle_info({'event', ['undefined' | Props]}, #state{node=Node}=State) ->
     {'noreply', State};
 handle_info({'event', [CallId | Props]}, #state{node=Node, publish_participant_event=EventsToPublish}=State) ->
     Action = props:get_value(<<"Action">>, Props),
+    lager:error("PROPS:~p", [Props]),
```

Comment:
@jamhed @fenollp there's actually a `props:to_log(Props, <<"THIS IS MYDEBUG">>)` that i find useful while developing